demo/jfx/src/main/java/com/acrolinx/client/sidebar/demo/jfx/AcrolinxJFXIntegration.java

```diff
@@ -6,7 +6,7 @@
 
 import javafx.scene.control.TextArea;
 
-import org.apache.commons.lang.NotImplementedException;
+import org.apache.commons.lang3.NotImplementedException;
```

Comment:
Depending on commons-lang just to throw this NotImplementedException is unnecessarily complex.
I suggest we throw java.lang.UnsupportedOperationException instead, then this class does not need to depend on Apache commons at all.

Same for the AcrolinxSwingIntegration class below.

Reply:
changed as requested